frontend/src/routes/(authenticated)/admin/UserFilter.svelte

```diff
@@ -0,0 +1,77 @@
+<script  context="module" lang="ts">
+  import type { User } from './+page';
+
+  export type UserFilter = {
```

Comment:
It's not a good idea to have a named export (the type `UserFilter`) with the same name as the name of the component (`UserFilter`). If you wanted to import this type and the component into the admin dashboard, you'd have to rename one of them in the import statement, e.g. `import UserFilter, { type UserFilter as UserFilterParams }` or something, which is poor DX (Developer eXperience) for whoever is using the component. I suggest doing the same thing that the ProjectFilter component did, and naming the type with a plural (there's a ProjectFitler component and a ProjectFilters type, so this type should be renamed to UserFilters).